assets/app/styles/_components.less

```diff
@@ -83,6 +83,7 @@
           overflow: hidden;
           text-overflow: ellipsis;
           line-height: 1.3;
+          white-space: nowrap;
```

Comment:
hmm, I see the issue also references other places that wrap awkwardly, but we still can't just set those to never wrap or we'll start truncating on longer image names, etc